frontend/src/components/UserSignUp/SignUpForm.jsx

```diff
@@ -234,6 +234,22 @@ function SignUpForm({ selectedEvent, handleModalClose, isAdult, setUser }) {
   // routing
   const history = useHistory();
 
+  const sendEmail = (data) => {
+    fetch("mail/register", {
+      method: "POST",
+      headers: {
+        Accept: "application/json",
+        "Content-Type": "application/json",
+      },
```

Comment:
I think this is duplicate code from the WaiverPage

Reply:
Yep, just refactored!